BoxBrowseSDK/BoxBrowseSDK/Controllers/BOXItemsViewController.m

```diff
@@ -202,7 +206,9 @@ - (void)refresh
                 self.items = items;
                 [self.tableView reloadData];
             }
-            [self.refreshControl endRefreshing];
+            if (self.refreshControl) {
```

Comment:
I'm not against it, but there's actually no need to add this check. Messaging `nil` is always okay in Objective-C, unless you're expecting a struct back (I can explain the technical details of that if you're interested).

Reply:
Ah... I *thought* that might have been the case. And meant to look deeper. Thanks for the background! :) I'll remove the check.